File: Java/01 - Introduction to Programming/Task06/numbers.java
import java.util.*;
public class numbers
{
  public static void main(String[] args)
  {
    Scanner keyboardInput = new Scanner(System.in);

    //request input from user
    System.out.print("Please enter an integer for num1: ");
    int num1 = Integer.parseInt(keyboardInput.nextLine());
    System.out.print("Please enter an integer for num2: ");
    int num2 = Integer.parseInt(keyboardInput.nextLine());
    System.out.print("Please enter an integer for num3: ");
    int num3 = Integer.parseInt(keyboardInput.nextLine());
       
    //sum of all numbers
    System.out.println(num1+num2+num3);

    //num1 minus num2
    System.out.println(num1-num2);

    //num3 multiplied by num1
    System.out.println(num3*num1);

    //sum of all three numbers divided by num3
    System.out.println((num1+num2+num3)/num3);

    keyboardInput.close();

  }
}

File: Ruby/01 - Introduction to Programming/Task06/numbers.rb
#Task6 - numbers.rb
#------------------------------------------------------------#
#request input from user
print "Please enter an integer for num1: "
num1 = gets.chomp().to_i
print "Please enter an integer for num2: "
num2 = gets.chomp().to_i
print "Please enter an integer for num3: "
num3 = gets.chomp().to_i

#sum of all numbers
puts(num1+num2+num3)

#num1 minus num2
puts(num1-num2)

#num3 multiplied by num1
puts(num3*num1)

#sum of all three numbers divided by num3
puts((num1+num2+num3)/num3)